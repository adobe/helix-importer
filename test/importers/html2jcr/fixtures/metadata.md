## Lorem ipsum

Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. A pellentesque sit amet porttitor eget dolor. Pellentesque sit amet porttitor eget dolor morbi non arcu risus. Id nibh tortor id aliquet lectus proin. Purus gravida quis blandit turpis cursus in hac habitasse platea. Amet tellus cras adipiscing enim eu turpis egestas pretium. Est sit amet facilisis magna. Venenatis urna cursus eget nunc scelerisque viverra mauris in aliquam.

+----------------------------------------------------+
| Metadata                                           |
+===============+====================================+
| Template      | article-template                   |
+---------------+------------------------------------+
| Title         | Sites Franklin Example             |
+---------------+------------------------------------+
| Description   | Test Description                   |
+---------------+------------------------------------+
| Abstract      | Lorem & Ipsum                      |
+---------------+------------------------------------+