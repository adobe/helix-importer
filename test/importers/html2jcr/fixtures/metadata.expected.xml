<?xml version="1.0" encoding="UTF-8"?>
<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0" jcr:primaryType="cq:Page">
    <jcr:content cq:template="/libs/core/franklin/templates/page" jcr:primaryType="cq:PageContent" sling:resourceType="core/franklin/components/page/v1/page" jcr:title="Sites Franklin Example" template="article-template" jcr:description="Test Description" abstract="Lorem &amp; Ipsum">
        <root jcr:primaryType="nt:unstructured" sling:resourceType="core/franklin/components/root/v1/root">
            <section sling:resourceType="core/franklin/components/section/v1/section" jcr:primaryType="nt:unstructured">
                <title sling:resourceType="core/franklin/components/title/v1/title" jcr:primaryType="nt:unstructured" jcr:title="Lorem ipsum" type="h2"/>
                <text sling:resourceType="core/franklin/components/text/v1/text" jcr:primaryType="nt:unstructured" text="&lt;p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. A pellentesque sit amet porttitor eget dolor. Pellentesque sit amet porttitor eget dolor morbi non arcu risus. Id nibh tortor id aliquet lectus proin. Purus gravida quis blandit turpis cursus in hac habitasse platea. Amet tellus cras adipiscing enim eu turpis egestas pretium. Est sit amet facilisis magna. Venenatis urna cursus eget nunc scelerisque viverra mauris in aliquam.&lt;/p>"/>
            </section>
        </root>
    </jcr:content>
</jcr:root>